exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -262,14 +262,11 @@ def _translate_data(
         pass
 
     def _export(self, data: TypingSequence[SDKDataT]) -> ExportResultT:
+
+        max_value = 64
```

Comment:
Would it be possible to make this configurable?  For example as an `export_timeout: maximum duration for exporting spans` argument to the exporter?

My use case is CLI applications where anything having them hang on exit for 64 seconds because a telemetry backend is down is unacceptable.   For comparison the Go implementation has [two separate timeouts](https://github.com/open-telemetry/opentelemetry-go/blob/cbcd4b1a3dc6ff839e06749c83294260aa671e6f/sdk/trace/batch_span_processor.go#L42-L50) `ExportTimeout` for the full operation and `BatchTimeout` for the direct publish.

For now I have special subclass with a copy of this method to tweak the behavior but it's a maintenance issue.

Reply:
I will bring up the topic of supporting configurability in next SIG meeting. Spec is not clear(at this point) in that regard and anything we do might become breaking change in future. And speaking of `ExportTimeout` and `BatchTimeout` python also has them as `schedule_delay_millis` and `export_timeout_millis` [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py#L142-L155) I don't know why but we are only enforcing the export timeout only during force flush.

Reply:
Talked about this in SIG call. This is out of scope for this. Consensus was this is not clear from Spec so we might end up making backwards incompatible change by introducing the support for configuring this. One thing we want to explore is making retry async and not blocking and also looking at how span processor timeout is behaving. Please create another issue and we will track it there.